src/cross.fs

```diff
@@ -107,6 +107,10 @@ variable xstate
   parse-next-name find-xname ?dup if >xcode else -1 abort" Unknown word " then ;
 
 : x:
-  xcreate x] ;
```

Comment:
Can we remove `xcreate` entirely?

Reply:
Maybe. I would keep it for a moment. It's just a cross version of `create`. It could be useful for `constant`, `variable`..  or just for the user.